### ISSUE TITLE
Feature request. Assembly (arm) support.

### ISSUE THREAD

**Issue by Wjxfi** [SpartanJ/ecode#78]
ARM processor
And more others

'

And support for everything on this site. 

https://www.dooseep.com/2023/01/assembly-language-and-architecture.html?m=1

**Comment by SpartanJ** [comment]
I don't think this is gonna happen. x86 ASM is more than enough for the moment. If you are interested in having ARM ASM support please consider collaborating. Thanks

**Comment by SpartanJ** [comment]
I won't close the issue. Added `help wanted` tag.
